tests/integration_tests/datasource_tests.py

```diff
@@ -543,7 +543,7 @@ def test_get_samples_with_filters(test_client, login_as_admin, virtual_dataset):
         f"/datasource/samples?datasource_id={virtual_dataset.id}&datasource_type=table"
     )
     rv = test_client.post(uri, json=None)
-    assert rv.status_code == 200
+    assert rv.status_code == 400
```

Comment:
why did this changed to a 400 now?

Reply:
Werkzeug 2.1.0 changelog:

> Request.get_json() will raise a 400 BadRequest error if the Content-Type header is not application/json. This makes a very common source of confusion more visible. https://github.com/pallets/werkzeug/issues/2339

If there's no json param or `json=None` then 400 http status will be thrown